src/Common/Commands.Common/Commands.Common.csproj

```diff
@@ -103,12 +103,16 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\packages\Microsoft.WindowsAzure.Management.4.1.1\lib\net40\Microsoft.WindowsAzure.Management.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.WindowsAzure.Storage">
```

Comment:
Looks like you've added a new DLL to the output. Please regenerate WIX file if that is the case